Backpack-SwiftUI/ImageGalleryGrid/Classes/ImageGalleryGridCategoriesCarousel.swift

```diff
@@ -0,0 +1,134 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+struct ImageGalleryGridCategoriesCarousel<ImageView: View>: View {
+    let categories: BPKImageGalleryCategoriesTypeContainer<ImageView>
+    @Binding var selectedCategory: Int?
+    private let categoryImageSideLength: CGFloat = 90
+    
+    var body: some View {
+        
+        switch categories {
+            
+        case .chip(let chipCategories):
```

Comment:
2nd smallest nit: you can just name the categories here `categories`, we know they're chip categories because we're inside the chip case